.ghci

```diff
@@ -1,4 +1,5 @@
 :set -hide-package=ghc-boot-th
+:set -package=ghc-lib-parser
```

Comment:
Why did this change now?

Reply:
I didn't make that change. Neil did I guess. Could it relate to the fact that `exposed=False` now?

Reply:
![Screen Shot 2020-01-09 at 12 55 51 PM](https://user-images.githubusercontent.com/41306216/72091970-657ce080-32df-11ea-8b17-e577079f3734.png)

Reply:
ah yeah that’s it, thanks for the explanation!